core/app/models/spree/variant.rb

```diff
@@ -56,6 +56,12 @@ class Variant < Spree::Base
     after_touch :clear_in_stock_cache
 
     scope :in_stock, -> { joins(:stock_items).where('count_on_hand > ? OR track_inventory = ?', 0, false) }
+    scope :in_stock_or_backorderable, -> do
+      joins(:stock_items).where('count_on_hand > ? OR
+                                 track_inventory = ? OR
```

Comment:
Don't use `?` inside query. Params are not dynamic and you can just pass them there

Reply:
@dimon-k you can just use ActiveRecord's `or` method

https://blog.bigbinary.com/2016/05/30/rails-5-adds-or-support-in-active-record.html

Reply:
Fixed.